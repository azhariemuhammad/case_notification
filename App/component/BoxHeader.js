import React from 'react'
import {View, Text, StyleSheet} from 'react-native'

const BoxHeader = (props) => {

    return (
         <View style={styles.flex1}>
             <Text style={styles.textCenter}>{props.title}</Text>
         </View>
    )
}

const styles = StyleSheet.create({
    flex1: {
        backgroundColor: '#F3F9FB',
        height: 92,
        justifyContent: 'center'
    },
    textCenter: {
        textAlign: 'center'
    }
})

export default BoxHeader
