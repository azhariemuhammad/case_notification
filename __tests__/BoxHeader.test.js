import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';

import BoxHeader from "../App/component/BoxHeader";



it("should render BoxHeader component and compare with snapshot", () => {
    const val = "Dinas Kesehatan Buru"
    const wrapper = shallow(<BoxHeader title={val}/>)
    expect(wrapper).toMatchSnapshot()
});
