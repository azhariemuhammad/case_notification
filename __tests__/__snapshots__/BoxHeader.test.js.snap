// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BoxHeader component and compare with snapshot 1`] = `ShallowWrapper {}`;
